test/io/test_io.py

```diff
@@ -1830,3 +1830,23 @@ def test_log_pool_req_observation(level, defaultenv):
         else:
             output = postgrest.read_stdout(nlines=4)
             assert len(output) == 0
+
+
+def test_proxy_status_header(defaultenv, metapostgrest):
+    "Test Proxy-Status header in statement timeout error"
```

Comment:
Are [PostgREST error codes](https://docs.postgrest.org/en/v12/references/errors.html#postgrest-error-codes) also exposed on the `Proxy-Status`? If so, we'd need some tests for them.

Reply:
I have currently exposed [PostgreSQL errors](https://docs.postgrest.org/en/v12/references/errors.html#errors-from-postgresql) only. I am not sure about the `pgrst` errors, do we need them to be exposed as well?

Reply:
Yes, all error codes should be exposed. Otherwise it would be inconsistent.

Reply:
Hmm, not all errors have a code which goes against this feature. I guess we would need a refactor first to have a code associated with all errors.

Reply:
I thought all of our errors had a code :thinking: . Yes, if you prefer open another issue/PR to clarify that.